.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
```

Comment:
Add a section here with guidelines on the following? 

- Refer to FAQ first, to see if this covered (link to faq)
- Join the mailing list to engage in conversations and get faster support.(link to signing up)
- If you are have triaged this as a real issue, then file a JIRA directly instead (link to instructions)